docs/faq/help-center.md

```diff
@@ -0,0 +1,30 @@
+---
+sidebar_label: 'Help Center'
+sidebar_position: 5
+---
+
+# Help Center
+
+### How to stay up to date about LUKSO?
+
+You can follow us on [Twitter](https://twitter.com/lukso_io), read our [Medium](https://medium.com/lukso) articles, and follow the announcements on our [Discord](https://discord.gg/lukso) server.
+
+### Where can I read LUKSOs Whitepaper?
+
+Here is the link to read and download the [Official LUKSO Whitepaper](https://uploads-ssl.webflow.com/629f44560745074760731ba4/62b200bfe0af12186845519a_LUKSO_Whitepaper_V1-1.pdf).
+
+### My question is not here. Where can I ask it?
+
+If you can't find an answer to your question, you can head to our [Discord](https://discord.gg/lukso) server. We have development-related channels where you can ask technical questions. We will add the most asked question to this list.
+
+### Where can I report regular issues for repositories and tools?
+
+We are preparing workflows to handle issues, investigations, and bug bounties. For now, please create an issue on our global [GitHub Issue Tracker](https://github.com/lukso-network/issue-tracker).
+
+### Where can security issues be reported to?
+
+Suppose it's a vulnerability or something that should not be shared with the public. Please reach out to official team members within our developer channel on [Discord](https://discord.gg/lukso).
+
+### Does LUKSO have grants or bug bounties?
+
+We're happy to compensate if there are severe findings and vulnerabilities discovered. The compensation will depend on the scope of the error or bug. We do not share precise bounty amounts beforehand because of the wide range of possible discoveries.
```

Comment:
We need to check this info with the business team before hand.
Please check with Jonathan.

Reply:
I have checked this with Fabian directly